packages/frontend/src/components/CreateSwap/CreateSwap.tsx

```diff
@@ -148,8 +156,8 @@ const CreateSwap = () => {
 
   const fromTokenKey = SwapFormKeyHelper.getTokenKey('from');
   const toTokenKey = SwapFormKeyHelper.getTokenKey('to');
-  const selectedFromToken = getTokenBySymbol(watch(fromTokenKey), tokens) || undefined;
-  const selectedToToken = getTokenBySymbol(watch(toTokenKey), tokens) || undefined;
+  const selectedFromToken = getTokenBySymbol(watch(fromTokenKey), fromTokens) || undefined;
```

Comment:
Can we get `fromToken` from `useSwapFormValues` instead of using `watch(fromTokenKey)`?

Reply:
Fixed in efa97d3b0ed28c9350e07f8b1b40d90ead20bbb9